### DISCUSSION TITLE
v1.0 🎉

### DISCUSSION THREAD

**Discussion by mattinannt** [formbricks/formbricks#558] [Announcements]
We are very proud to introduce Formbricks v1.0 🎉

## What's Changed

* 📝 Add new Filter-Options in Summary / Responses view by @gitstart-formbricks in https://github.com/formbricks/formbricks/pull/474
* 📤 Export to CSV by @pandeymangg in https://github.com/formbricks/formbricks/pull/371
* 🖊️ Prefill First Question by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/376
* 🗄️ Add ability to archive attributes by @Ashutosh-Bhadauriya in https://github.com/formbricks/formbricks/pull/365
* 👥 Add user roles to invite and allow changing roles by @neilchauhan2 in https://github.com/formbricks/formbricks/pull/404
* 📝 Feature to add a note to responses by @gitstart-formbricks in https://github.com/formbricks/formbricks/pull/344
* 🌐 Add Webhooks by @mattinannt in https://github.com/formbricks/formbricks/pull/364
* 🎲 Add option to randomize choices in multiple choice questions by @Shubhdeep12 in https://github.com/formbricks/formbricks/pull/394
* 🚀 Add option to redirect to a new page after a user finishes a survey by @Pradumn27 in https://github.com/formbricks/formbricks/pull/464
* 🎨 Revamp question card ui by @Ashutosh-Bhadauriya in https://github.com/formbricks/formbricks/pull/351
* 👌 Improve DX with new response data model + services + client endpoints by @mattinannt in https://github.com/formbricks/formbricks/pull/355
* 🗓️ Add close-on-date functionality to surveys by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/463
* 🔀 add: move-from-prod-to-dev-rev by @Ankur-Datta-4 in https://github.com/formbricks/formbricks/pull/392
* ⚠️ Add unsaved changes reminder to survey editor by @joe-shajan in https://github.com/formbricks/formbricks/pull/409
* 🎫 Show current user attributes in response by @Pradumn27 in https://github.com/formbricks/formbricks/pull/403
* 🖼️ Add iframe embed option for link surveys 402 by @Ankur-Datta-4 in https://github.com/formbricks/formbricks/pull/414
* 📋 Show active and inactive surveys in attribute list by @joe-shajan in https://github.com/formbricks/formbricks/pull/400
* 🔄 Revamped our Demo App for trying formbricks-js features by @jobenjada in https://github.com/formbricks/formbricks/pull/372
* 🆕 Enhance: Add save button in deleteDialogue when creating survey by @Ashutosh-Bhadauriya in https://github.com/formbricks/formbricks/pull/377
* 🚀 Add Integrations Page by @mattinannt in https://github.com/formbricks/formbricks/pull/439
* 🎬 Add generic actions "90% scroll" & "exit intent" by @Ashutosh-Bhadauriya in https://github.com/formbricks/formbricks/pull/393
* 🏷️ Add response tagging functionality @pandeymangg in https://github.com/formbricks/formbricks/pull/426
* 💻 Add user agent to response meta by @G3root in https://github.com/formbricks/formbricks/pull/398
* ↪️ Add feature to redirect on survey completion by @Dhruwang in https://github.com/formbricks/formbricks/pull/408
* 🔒 Add Formbricks Login Password validation by @Dhruwang in https://github.com/formbricks/formbricks/pull/412
* 🔗 Add option to add userId to link survey  #384 by @Zorig in https://github.com/formbricks/formbricks/pull/430
* 🎯 Option to position in-app survey by @moritzrengert in https://github.com/formbricks/formbricks/pull/440
* 🌍 Add env variable NEXT_PUBLIC_INVITE_DISABLED to allow or not for team invite by @niteshseram in https://github.com/formbricks/formbricks/pull/373
* 🐳 Add Public Docker Image for Formbricks to DockerHub by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/546
* 📅 Add weekly summary feature by @singhbhaskar in https://github.com/formbricks/formbricks/pull/431
* 📎 Add option to share team invite link by @ty-kerr in https://github.com/formbricks/formbricks/pull/452
* 🗑️ Add delete user account feature by @fdis111 in https://github.com/formbricks/formbricks/pull/363
* 🏁 Mark survey as completed after x responses by @surendhar-palanisamy in https://github.com/formbricks/formbricks/pull/395
* 🌐 Improve Page Titles by @Dhruwang in https://github.com/formbricks/formbricks/pull/470
* 📤 Add new Surveys endpoint to list all surveys of an account by @mattinannt in https://github.com/formbricks/formbricks/pull/500
* 🧪 DX: Add Megaprompt for improved chatGPT Dev Experience by @mattinannt in https://github.com/formbricks/formbricks/pull/381
* 🎯 DX: Add pnpm go command by @Dhruwang in https://github.com/formbricks/formbricks/pull/538
* 🧪 Add Testing Suite for JS library by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/399
* 🔧 Refactor Survey Templates Page by @Ankur-Datta-4 in https://github.com/formbricks/formbricks/pull/488
* ☁️ Formbricks-js now sync automatically with the backend/cloud by @mattinannt in https://github.com/formbricks/formbricks/pull/537

## Bugfixes
* 🐛 Close Icon not showing on mobile in formbricks-js by @mattinannt in [https://github.com/formbricks/formbricks/pull/382](https://github.com/formbricks/formbricks/pull/382)
* 🐞 Fix email notifications display empty responses, fix webhook data format by @mattinannt in [https://github.com/formbricks/formbricks/pull/383](https://github.com/formbricks/formbricks/pull/383)
* 📜 Preserve docs nav scroll position by @joe-shajan in [https://github.com/formbricks/formbricks/pull/380](https://github.com/formbricks/formbricks/pull/380)
* 🚫 Deleted responses are no longer shown in frontend by @mattinannt in [https://github.com/formbricks/formbricks/pull/436](https://github.com/formbricks/formbricks/pull/436)
* 🔒 Improve Authorization Checks in Layout by @mattinannt in [https://github.com/formbricks/formbricks/pull/487](https://github.com/formbricks/formbricks/pull/487)
* 🔄 Fix error during onboarding when environment not fully loaded by @mattinannt in [https://github.com/formbricks/formbricks/pull/459](https://github.com/formbricks/formbricks/pull/459)
* ⚠️ Fix the condition for 0 response in survey menubar by @niteshseram in [https://github.com/formbricks/formbricks/pull/553](https://github.com/formbricks/formbricks/pull/553)
* 🖇️ Fix "other" link option by @moritzrengert in [https://github.com/formbricks/formbricks/pull/502](https://github.com/formbricks/formbricks/pull/502)
* 💌 Fix invites not working sometimes by @moritzrengert in [https://github.com/formbricks/formbricks/pull/475](https://github.com/formbricks/formbricks/pull/475)
* 📚 Improve documentation by @jobenjada in [https://github.com/formbricks/formbricks/pull/526](https://github.com/formbricks/formbricks/pull/526)
* 🎛️ QuestionIds not updating in existing logic by @Dhruwang in [https://github.com/formbricks/formbricks/pull/530](https://github.com/formbricks/formbricks/pull/530)
* 🔒 Fix admin owner permission by @moritzrengert in [https://github.com/formbricks/formbricks/pull/539](https://github.com/formbricks/formbricks/pull/539)
* 📧 Fix email notification not working under certain conditions by @mattinannt in [https://github.com/formbricks/formbricks/pull/531](https://github.com/formbricks/formbricks/pull/531)
* 🛠️ Fix error while saving in-app surveys under certain conditions by @mattinannt in [https://github.com/formbricks/formbricks/pull/532](https://github.com/formbricks/formbricks/pull/532)
* 📩 Fix team invite not working with github login by @moritzrengert in [https://github.com/formbricks/formbricks/pull/534](https://github.com/formbricks/formbricks/pull/534)
* 📝 Fix long description overflow by @niteshseram in [https://github.com/formbricks/formbricks/pull/466](https://github.com/formbricks/formbricks/pull/466)
* 🛣️ Fix Roadmap Template productName replacement not working by @jobenjada in [https://github.com/formbricks/formbricks/pull/535](https://github.com/formbricks/formbricks/pull/535)
* 🚫 Prevent duplicate team invite by @moritzrengert in [https://github.com/formbricks/formbricks/pull/536](https://github.com/formbricks/formbricks/pull/536)

## New Contributors
* 🎉 @Ashutosh-Bhadauriya made their first contribution in [https://github.com/formbricks/formbricks/pull/351](https://github.com/formbricks/formbricks/pull/351)
* 🎉 @gitstart-formbricks made their first contribution in [https://github.com/formbricks/formbricks/pull/344](https://github.com/formbricks/formbricks/pull/344)
* 🎉 @pandeymangg made their first contribution in [https://github.com/formbricks/formbricks/pull/371](https://github.com/formbricks/formbricks/pull/371)
* 🎉 @joe-shajan made their first contribution in [https://github.com/formbricks/formbricks/pull/380](https://github.com/formbricks/formbricks/pull/380)
* 🎉 @gupta-piyush19 made their first contribution in [https://github.com/formbricks/formbricks/pull/376](https://github.com/formbricks/formbricks/pull/376)
* 🎉 @surendhar-palanisamy made their first contribution in [https://github.com/formbricks/formbricks/pull/395](https://github.com/formbricks/formbricks/pull/395)
* 🎉 @neilchauhan2 made their first contribution in [https://github.com/formbricks/formbricks/pull/404](https://github.com/formbricks/formbricks/pull/404)
* 🎉 @Ankur-Datta-4 made their first contribution in [https://github.com/formbricks/formbricks/pull/414](https://github.com/formbricks/formbricks/pull/414)
* 🎉 @niteshseram made their first contribution in [https://github.com/formbricks/formbricks/pull/373](https://github.com/formbricks/formbricks/pull/373)
* 🎉 @Pradumn27 made their first contribution in [https://github.com/formbricks/formbricks/pull/403](https://github.com/formbricks/formbricks/pull/403)
* 🎉 @G3root made their first contribution in [https://github.com/formbricks/formbricks/pull/398](https://github.com/formbricks/formbricks/pull/398)
* 🎉 @Dhruwang made their first contribution in [https://github.com/formbricks/formbricks/pull/408](https://github.com/formbricks/formbricks/pull/408)
* 🎉 @Zorig made their first contribution in [https://github.com/formbricks/formbricks/pull/430](https://github.com/formbricks/formbricks/pull/430)
* 🎉 @PeerRich made their first contribution in [https://github.com/formbricks/formbricks/pull/448](https://github.com/formbricks/formbricks/pull/448)
* 🎉 @ShubhamPalriwala made their first contribution in [https://github.com/formbricks/formbricks/pull/399](https://github.com/formbricks/formbricks/pull/399)
* 🎉 @ty-kerr made their first contribution in [https://github.com/formbricks/formbricks/pull/452](https://github.com/formbricks/formbricks/pull/452)
* 🎉 @singhbhaskar made their first contribution in [https://github.com/formbricks/formbricks/pull/431](https://github.com/formbricks/formbricks/pull/431)
* 🎉 @subh-cs made their first contribution in [https://github.com/formbricks/formbricks/pull/432](https://github.com/formbricks/formbricks/pull/432)
* 🎉 @Meetcpatel made their first contribution in [https://github.com/formbricks/formbricks/pull/489](https://github.com/formbricks/formbricks/pull/489)

Thanks a lot to all these amazing contributors who made this release possible in such a short amount of time 💪🚀🙏

**Full Changelog**: https://github.com/formbricks/formbricks/compare/v0.15...v1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/formbricks/formbricks/releases/tag/v1.0'>v1.0 🎉</a>.</em>

**Comment by Dhruwang** [comment]
🚀🥳
